data/newMigrations/007-service_type_providers.js

```diff
@@ -0,0 +1,26 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('service_type_providers', (tbl) => {
+        tbl.increments('service_type_provider_id').primary();
+        tbl
+        .integer('service_type_id')
+        .unsigned()
+        .notNullable()
+        .references('service_type_id')
+        .inTable('service_types')
+        .onUpdate('CASCADE')
+        .onDelete('CASCADE');
+        tbl
+        .string('provider_id')
+        .unsigned()
+        .notNullable()
+        .references('provider_id')
+        .inTable('providers')
+        .onUpdate('CASCADE')
+        .onDelete('CASCADE');
```

Comment:
I'm not sure we should be using CASCADE for onDelete or onUpdate. We should probably switch this to RESTRICT